src/main.ts

```diff
@@ -100,7 +100,7 @@ if (program.host || configuration?.host) {
     program.host || configuration?.host || "localhost",
     program.user || configuration?.user || "root", // bad practice but unfortunately common, make it easier for the user
     program.password || configuration?.password,
-    program.port || configuration?.port || "3306"
+    program.port || configuration?.port || undefined
```

Comment:
Why no default fallback?

Reply:
`mysql2` and `pg` both fall back to their default ports if it's undefined. The default port for postgres is `5432`, not `3306` 😔

Reply:
Okay so leaving this as undefined is better then I guess? Noice

Reply:
Do you want me to explicitly define the default port for the driver in main.ts instead?

Reply:
I like what you've got leave it as is :+1: